README.md

```diff
@@ -17,6 +17,35 @@ To install BenchmarkTools, you can run the following:
 Pkg.add("BenchmarkTools")
 ```
 
+## Quick Start
+
+The simplest usage is via the `@btime` macro, which is analogous to Julia's built-in [`@time` macro](https://docs.julialang.org/en/stable/stdlib/base/#Base.@time) but is often more accurate (by collecting results over multiple runs):
+
+```julia
+julia> using BenchmarkTools, Compat   # you need to use both modules
+
+julia> @btime sin(1)
+  15.081 ns (0 allocations: 0 bytes)
+0.8414709848078965
+```
+
+If the expression you want to benchmark depends on global variables, you should use `$` to "interpolate" them into the benchmark expression to [avoid the problems of benchmarking with globals](https://docs.julialang.org/en/stable/manual/performance-tips/#Avoid-global-variables-1).  Essentially, any interpolated variable `$x` or expression `$(...)` is "pre-computed" before benchmarking begins:
```

Comment:
> If the expression you want to benchmark depends on global variables

Might be more clear for naive users if we change `global` to `external` here. 

Also might be a good idea to link to https://github.com/JuliaCI/BenchmarkTools.jl/blob/master/doc/manual.md#interpolating-values-into-benchmark-expressions from here.

Reply:
Okay, will do.